docs/website/docs/general-usage/credentials/configuration.md

```diff
@@ -0,0 +1,424 @@
+---
+title: Secrets and Configs
+description: Overview secrets and configs
+keywords: [credentials, secrets.toml, secrets, config, configuration, environment
+      variables]
+---
+
+# Secrets and Configs
+
+## General Usage and an Example
+
+This is what we had in mind, when designing:
```

Comment:
Here's also the instructions need to be removed

Reply:
it's a part of text :D

Reply:
Ok, I get it, this part still needs to be rephrased for better comprehension. This is an opening sentence in a documentation about creds. I'll leave some comments about this above